### ISSUE TITLE
[Suggestion] Increase max distance based on rarity

### ISSUE THREAD

**Issue by ryanball** [timwah/pokeslack#7]
Let's say a Legendary spawns a mile away. I'm going to be willing to run for that pokemon so it should get more distance priority before being filtered out.

**Comment by timwah** [comment]
I agree, checkout the [wiki](https://github.com/timwah/pokeslack/wiki) for the roadmap.
- Configure RARITY_LIMIT as a tuple of rarities and distances, like [(1, 0.2), (2, 0.4), (3, 0.6),...], i.e. for more rare Pokemon you're willing to travel and get notified at longer distances

That's one idea for the implementation, let me know if you have another suggestion. Thanks!

**Comment by ryanball** [comment]
Perfect! I'll think of ideas if I have any, but that seems like a reasonable starting point
